contracts/FlowIDTableStaking.cdc

```diff
@@ -765,6 +766,14 @@ pub contract FlowIDTableStaking {
             return <-node
         }
 
+        pub fun setNodeWeight(nodeRecord: &NodeRecord, weight: UInt64) {
```

Comment:
I don't like the idea of the admin having a reference to a node record outside of the contract, so I'd rather the signature of this be `setNodeWeight(nodeID: String, weight: UInt64)`, then borrow the node record in the function and modify it there

Reply:
Sure. That's an easy enough change and I'll do it. I'm wondering if you could elaborate a little for my benefit on the risk with the reference.

Reply:
Having the node reference outside of the contract allows the admin to have access to all of the nodes' tokens, which we don't want. I'm not even sure how you were expecting the reference to the `NodeRecord` to be returned from the contract anyway since the function that borrows it is`access(account)`